src/main/java/core/basesyntax/thread/MyThread.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax.thread;
 
-public class MyThread {
-    // write your code here
+public class MyThread extends Thread {
+    @Override
+    public void run() {
+        int i = 0;
+        while (i < 21) {
+            System.out.println(i);
+            i++;
+        }
```

Comment:
```suggestion
        for (int i = 0; i < 21; i++) {
            System.out.println(i);
        }
```

I think it looks easier this way